mkt/ratings/tests/test_models.py

```diff
@@ -38,3 +38,15 @@ def test_review_aggregates_triggers_reindex(self, index_webapps):
         index_webapps.reset_mock()
         addon_review_aggregates(self.app.pk)
         assert index_webapps.called
+
+    def test_soft_delete(self):
+        eq_(Review.objects.count(), 2)
+        eq_(Review.with_deleted.count(), 2)
```

Comment:
These two lines just show you the fixtures set up correctly, I don't think they contribute to the test.